test/basic.html

```diff
@@ -469,7 +516,9 @@
 
         // Wait one tick for observeNodes.
         Polymer.Base.async(function() {
-          form._form.querySelector('input[type=submit]').click();
+          form.$.content.assignedNodes()
+            .find((n) => n.nodeType === Node.ELEMENT_NODE)
```

Comment:
Please don't use es6 code -- this is a hybrid element, which needs to work with Polymer 1, which is used in browsers that do not support es6